Common/src/config_structure.cpp

```diff
@@ -1541,6 +1544,8 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   addBoolOption("WRT_SHARPEDGES", Wrt_SharpEdges, false);
   /* DESCRIPTION: Output the rind layers in the solution files  \ingroup Config*/
   addBoolOption("WRT_HALO", Wrt_Halo, false);
+  /* DESCRIPTION: Output the performance summary to the console at the end of SU2_CFD  \ingroup Config*/
+  addBoolOption("WRT_PERFORMANCE", Wrt_Performance, false);
```

Comment:
This is useful, thanks!

Reply:
It's a first implementation that can certainly be moved and improved as part of the output overhaul. Looking forward to that!